test/e2e/features/step-definitions/room-list.js

```diff
@@ -11,7 +11,26 @@ defineSupportCode(({ Given, When, Then }) => {
     await driver.findElement(By.className('roomsInput')).click()
   })
 
-  Then('I see a list of rooms', async () => {
+  Then('I see a list of rooms and their availability', async () => {
     await driver.findElement(By.tagName('h3'))
+    const condition = until.elementLocated(By.tagName('p'))
+    const element = await driver.wait(condition)
+    await driver.wait(until.elementTextContains(element, 'Available'))
   })
+
+  // Given('I am on the list of rooms', async () => {
```

Comment:
Just to note: As we (Zac and I) were writing this cucumber test, we realized that it would fail the integration/staging build if pushed to master. I want some way to test the functionality of changing rooms, but since we don't know the state of the database at a given point in time, am unsure of the best way to approach that given different testing environments. Noting this to bookmark for future discussion, and to explain why this test is here but commented out.

Reply:
Yeah, the DB is always re-initialized when the API is built/deployed, but that doesn't always work for the front-end.  Our cheesy solution at my prior job was a "cleanup" service that was only deployed to testing environments.  Not sure how that would go over here.